app/views/checkouts/_due_date.html.erb

```diff
@@ -0,0 +1,3 @@
+<td class="due_date">
```

Comment:
You know what's kind of weird to me, that upon successful renew this `td` isn't duplicated. It seems to me like you target inner html on this td and replace it with a spinner, then on success it renders the new due date with this partial so there should be a nested td, but there isn't 🤷‍♂️

Reply:
yeah looking at it again now,I agree like how there isn't a nested td there, I do not understand 🤷

Reply:
I think we should change it so the surround `td` is outside the partial, who knows, browsers could change this seeming auto-correction fix that's present now.

Reply:
yup, I was just working on changing it :)

Reply:
updated